addons/web/static/src/views/list/list_controller.js

```diff
@@ -70,6 +71,7 @@ export class ListController extends Component {
             groupByInfo: this.archInfo.groupBy.fields,
             limit: this.archInfo.limit || this.props.limit,
             defaultOrder: this.archInfo.defaultOrder,
+            defaultGroupBy,
```

Comment:
```suggestion
            defaultGroupBy: this.props.defaultGroupBy,
```
Since we just use here the `defaultGroupBy` variable we could directly do that instead of creating a variable. 🙂